examples/script/core/2_conditions.py

```diff
@@ -28,7 +28,7 @@
 then the actor performs the corresponding transition.
```

Comment:
With the current tutorial reading order (`script/core/1_basics`, `script/core/2_conditions`, `pipeline/1_basics`) this actor mention is before actor explanation. I suggest either changing recommended tutorial reading order in #90 or removing this mention.